securedrop/models.py

```diff
@@ -102,10 +106,41 @@ def collection(self):
         collection.sort(key=lambda x: int(x.filename.split('-')[0]))
         return collection
 
+    @property
```

Comment:
Nit: We may want to set the `setter` and `deleter` attributes of this to be `NotImplementedError`.